packages/elements/src/components/ino-chip/ino-chip.tsx

```diff
@@ -26,7 +26,13 @@ export class Chip implements ComponentInterface {
   @Prop() inoFill?: ChipSurface = 'solid';
 
   /**
-   * If enabled, prepends the slotted icon to the chip
+   * Prepends an icon to the chip label.
+   * @deprecated This property is deprecated and will be removed with the next major release. Instead, use the ino-icon-leading slot.
```

Comment:
I would suggest that we also check if the `inoIcon` property is used initially (e.g. in `componentDidMount`) and print a warning with `console.warn` with the deprecated message above. This way we make sure that everyone is informed even if he's not checking the documentation regularly.

Reply:
Good idea